packages/shared-core/src/filters.ts

```diff
@@ -142,25 +142,6 @@ export function recurseLogicalOperators(
   return filters
 }
 
-export function recurseSearchFilters(
```

Comment:
Where did this move to?

Reply:
This usage was slightly wrong and only used in one place. This merged it and refactored it in `packages/server/src/threads/automation.ts` directly